newsplease/pipeline/pipelines.py

```diff
@@ -122,10 +122,91 @@ def close_spider(self, spider):
         # Close DB connection - garbage collection
         self.conn.close()
 
+class ExtractedInformationStorage(object):
+    """
+    Provides basic functionality for Storages
+    """
 
-class MySQLStorage(object):
+    log = None
+
+    def __init__(self):
+        self.log = logging.getLogger(__name__)
+        self.log.addHandler(logging.NullHandler())
+        self.cfg = CrawlerConfig.get_instance()
+
+    @staticmethod
+    def ensure_str(text):
+        if isinstance(text, str):
+            return text
+        else:
+            return text.decode('utf-8')
+
+    @staticmethod
+    def extract_relevant_info(item):
+        """
+        extracts from an item only fields that we want to output as extracted information
+        :rtype: object
+        :param item:
+        :return:
+        """
+        article = {
```

Comment:
Instead of this, could you use https://github.com/fhamborg/news-please/blob/master/newsplease/NewsArticle.py#L22 ?